src/transformDimension.ts

```diff
@@ -1,15 +1,22 @@
 import { DesignToken } from 'style-dictionary/types';
 
-function _transformDimension(dim: string | number): string {
-  // Check if the value is numeric with isNaN, this supports string values as well
-  // Check if value is not empty string, since this is also not considered "NaN"
-  // Check if the value, when parsed (since it can also be number), does not equal 0
+function ensurePxSuffix(dim: string | number): string {
   if (!isNaN(dim as number) && dim !== '' && parseFloat(dim as string) !== 0) {
     return `${dim}px`;
   }
   return `${dim}`;
 }
 
+function splitMultiIntoSingle(dim: string | number): string {
+  if (typeof dim === 'string' && dim.includes(' ')) {
+    return dim
+      .split(' ')
+      .map(val => ensurePxSuffix(val))
```

Comment:
```suggestion
      .map(ensurePxSuffix)
```
Callback functions can be shortened like that if the signature from the map matches or can be considered an extension of the signature of the callback function.

So in this case, map gives 3 arguments, "item", "index" and "fullArray", which is valid to pass directly into the ensurePxSuffix (it ignores the 2nd and 3rd argument as it does not have a param for it)

Nitpick though, up to you to change or not